README.md

```diff
@@ -1,27 +1,49 @@
+
 # Buzz About
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 8.0.0.
+[![Build Status][Angular Tests badge]][Angular Tests page]
+[![Deployment Status][Deploy badge]][Deploy page]
+
+</div align="center">
+
+
+Buzz About is a participatory simulation that teaches people about native pollinators of Minnesota.
+
+Buzz About is a web app built with [Angular][] and [Firebase][].
 
-## Development server
+## Usage
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+You can find Buzz About online at [buzzabout.app][].
 
-## Code scaffolding
+## Contributing
 
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+If you'd like to contribute, please fork the repository and make changes as you'd like. Pull requests are warmly welcome.
 
-## Build
+For more information on how to contribute to Buzz About, see [DEVELOPMENT.md][].
 
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build.
+## Code of Conduct
 
-## Running unit tests
+All repositories in the Minnesota Pollinators organization are subject to our [Code of Conduct][].
 
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+## Acknowledgements
 
-## Running end-to-end tests
+Funding for this project was provided by the Minnesota Environment and Natural Resources Trust Fund as recommended by the [Legislative-Citizen Commission on Minnesota Resources (LCCMR)][LCCMR].
 
-Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
+The Trust Fund is a permanent fund constitutionally established by the citizens of Minnesota to assist in the protection, conservation, preservation, and enhancement of the state’s air, water, land, fish, wildlife, and other natural resources. Currently 40% of net Minnesota State Lottery proceeds are dedicated to growing the Trust Fund and ensuring future benefits for Minnesota’s environment and natural resources.
 
-## Further help
 
-To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
+[Angular Tests badge]: ../../workflows/Angular%20Tests/badge.svg
+[Angular Tests page]: ../../actions?query=workflow%3A"Angular+Tests"
+[Deploy badge]: ../../workflows/Firebase%20Deploy/badge.svg
+[Deploy page]: ../../actions?query=workflow%3A"Firebase+Deploy"
+[Angular]: https://angular.io/
+[Firebase]: https://firebase.google.com/
+[AR Markers]: ./AR_Markers.pdf
+[buzzabout.app]: html://buzzabout.app
+[buzzabout.app/play]: html://buzzabout.app/play
```

Comment:
```suggestion
[buzzabout.app]: https://buzzabout.app/
[buzzabout.app/play]: https://buzzabout.app/play
```

Also, is there a reason we are using the reference style links for every link?

Reply:
Oops, good catch, thank you! :sheep:
Re: reference style links, not particularly, I just like keeping URLs out of running text :) That's just me, though